documentation/backup_restore.md

```diff
@@ -1,47 +1,59 @@
+* `custom`: Use this type when you want to selectively back up and restore data for specific services. 
 
-The restore needs to specify the backup file path to restore from in its `input_file` parameter.
+If no type is specified, the default is `recovery`.
 
-**Important note**: peadm backup and restore can only be used on PE installs which are created using PEADM. Also, you can only use `peadm::restore` on a backup created by `peadm::backup`.
+When backing up or restoring PE, you must use the `--targets` option to specify the hostname (FQDN) of your primary server.
+ 
+The backup file created is named `pe-backup-YYYY-MM-DDTHHMMSSZ.tar.gz` and placed by default in `/tmp`. To specify a different location for the backup file, you can define the `output_directory` parameter.
 
-A backup and restore need to specify the primary server as the target.
+This example shows how to run a `recovery` backup which places the backup file in a custom location.  
+```
+bolt plan run peadm::backup --targets my.primary.vm backup_type=recovery output_directory=/custom_path
+```
+
+When restoring PE, you must define the `input_file` parameter to specify the path to the backup file you want to use. For example:
+```
+bolt plan run peadm::restore --targets my.primary.vm input_file="/tmp/my_backup.tar.gz"
+```
 
-## Recovery backup type
+## Using `recovery` backup and restore
 
-When backup type `recovery` is selected, the primary backup created is intended to be used to recover the primary it if it fails. This means that the primary's configuration will be restored to exactly the state it was in when the backup was created.
+When you run a `recovery` backup plan, the primary server configuration is backed up in full. In the event of a primary server failure, this backup can be used to restore the PE installation to the state it was in when the backup was created.
 
-You can create a recovery backup as follows:
+You can create a `recovery` backup as follows:
 ```
 bolt plan run peadm::backup --targets my.primary.vm backup_type=recovery
 ```
-or, simply,
+Alternatively, because `recovery` is the default type, you can use this simplified command:
 ```
 bolt plan run peadm::backup --targets my.primary.vm
 ```
 
-To restore from this backup, you can do:
+To restore your installation from this backup, run:
 ```
 bolt plan run peadm::restore --targets my.primary.vm input_file="/tmp/my_backup.tar.gz"
 ```
 
-The recovery restore will also work if some or all services are down on the primary. The restore process will restart the services at the end.
+**Tip**: Restoring from a `recovery` backup restarts any services that are unavailable on the primary server.
 
-## Custom backup type
+## Using `custom` backup and restore
 
-When selecting the `custom` backup type, the user can choose which items are backed up and/or restored using the `backup` (or `restore`) plan parameter.
-This parameter can be omitted, in which case all options will be turned on by default.
```

Comment:
@J-Hunniford I think it's important to note that when using the `custom` backup type, all items are backed up by default, so not specifying the options will backup (and restore) everything. This means that you only need to specify items you don't want to be backed up.